### DISCUSSION TITLE
Max Token Limits Incorrect?

### DISCUSSION THREAD

**Discussion by lhyphendixon** [langgenius/dify#7243] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hey guys, for some reason many models in both the Dify Cloud and Self-Hosted instances have Max Token limits far short of their actual capabilities.  Surely I’m just missing something.  The Claude models, for example, should have 200k token capabilities, yet, when I’m adjusting the model options in the chatbot builder, it only lets me go up to 8k.  GPT-4 models are also well short of what they should be aside from the 32k model, which appropriately allows up to 32k.  What’s going on?

**Comment by crazywoola** [comment]
200k is context size, and 4k is the max tokens of output. I think you are referring to this one.

**Reply by zq99299** [reply]
#13451  It should be the same problem I encountered. The actual number of input tokens is only 9786, but Dify directly throws an error: "Model Parameter max_tokens should be less than or equal to 12288.0."
What exactly is the problem? How can I solve it?
